orca-core/src/main/java/com/netflix/spinnaker/orca/config/UserConfiguredUrlRestrictions.java

```diff
@@ -86,10 +94,9 @@ public URI validateURI(String url) throws IllegalArgumentException {
       if (!allowedSchemes.contains(u.getScheme().toLowerCase())) {
         throw new IllegalArgumentException("unsupported URI scheme " + url);
       }
-      if (!allowedHostnames.matcher(u.getHost()).matches()) {
-        throw new IllegalArgumentException("Host not allowed " + u.getHost() + ". Host much match " + allowedHostnames.toString() + ".");
+      if (!allowedHostnames.matcher(u.getHost()).matches() && !whitelistedDomains.contains(u.getHost())) {
```

Comment:
This approach is just a simpler version of adding to the regex, e.g.
`'^(.+\.netflix(?:aws)?\.(?:net|com)$)|(myawesomedomain.com)'

Reply:
I would be fine with just modifying the regex as-is in our case and putting off this PR until sometime in the future when our regex becomes unwieldy.

Reply:
Sounds good to me , i will close this PR for now.